compiler/src/Language/Bond/Codegen/Cpp/Util.hs

```diff
@@ -16,9 +16,6 @@ module Language.Bond.Codegen.Cpp.Util
     , attributeInit
     , schemaMetadata
     , ifndef
-    , defaultedFunctions
```

Comment:
This is a small breaking change (removal of exported definitions), so instead of the version change to 0.10.2.0, we need to go to 0.11.0.0.

Reply:
Fixed.